components/credentials/build.gradle

```diff
@@ -44,8 +44,8 @@ dependencies {
     implementation('org.springframework.boot:spring-boot-starter-security')
     implementation('org.springframework.boot:spring-boot-starter-validation')
     testImplementation("org.springframework.boot:spring-boot-starter-test")
-    testImplementation("junit:junit")
-
+    testImplementation(platform('org.junit:junit-bom:5.10.2'))
```

Comment:
Specifying version here is unnecessary. I will commit a change to address this myself.